vespa/package.py

```diff
@@ -130,6 +130,7 @@ def __init__(
         weight: Optional[int] = None,
         bolding: Optional[Literal[True]] = None,
         summary: Optional[Summary] = None,
+        query_command: Optional[List[str]] = None,
     ) -> None:
```

Comment:
This constructor is getting out of hand. I wonder if this should be changed to a `kwargs` or a simple dict?

Reply:
Yeah, it is filling up fast, I think we can have it accept a `TypedDict` that has `NotRequired[Type]` in all fields so you can pass a dictionary of what you want to configure.

Something like (writing off the top of my head):

```python3
class FieldConfigurationDict(TypedDict):
	weight: NotRequired[int]
	bolding: NotRequired[Literal[True]]
	query_command: NotRequired[List[str]]

class Field:
	def __init__(self, all, previous, constructors, to keep, backwards, compat, **kwargs: FieldConfigurationDict):
		"""Code that runs over every value and assigns it correctly"""
```